py/desispec/scripts/specex.py

```diff
@@ -237,6 +237,60 @@ def main(args, comm=None):
 
     return
 
+def run(comm,cmds,cameras):
+    """
+    Run PSF fits with specex on a set of ccd images 
+
+    Args:
+        comm:    MPI communicator 
+        cmds:    dictionary of command arguments to main method of this module
+        cameras: entries in cmds to be run in parallel, one entry per ccd image to be fit
```

Comment:
Documentation clarification requests:

`cmds` is a dictionary keyed by a camera string (e.g. 'b0', 'r1', ...) with values being the `desi_compute_psf ...` string that one would run on the command line?  i.e. not the cmdargs for main(cmdargs) themselves, nor the list that is passed to parse(...), right?

`cameras` is a list of keys in the `cmds` dict that should be processed?  It looks like you support the ability to pass `cameras` entries that aren't a key to `cmds`.  Is that an intended use-case, or should that be logged as an error (while still continuing with the others and not crashing)?

The `comm` here would normally be a "big" communicator that would be split into smaller sub-communicators to process individual cameras (with the limiting case of comm having 20 ranks and not getting split).  Are there any constraints on comm having an even multiple of 20 ranks?  Would it ever make sense to support a smaller group_size as an input argument for different packing?